.clang-format

```diff
@@ -0,0 +1,9 @@
+BasedOnStyle: Microsoft
+ColumnLimit: 120
+AllowShortIfStatementsOnASingleLine: WithoutElse
+SortIncludes: false
+PointerAlignment: Left
+AlignConsecutiveAssignments: Consecutive
+AlignAfterOpenBracket: BlockIndent
+AllowShortFunctionsOnASingleLine: Inline
+NamespaceIndentation: All
```

Comment:
@MiranDMC chime in if you think some rules should be added or changed. Be constructive.